source/shared/cs_SharedConfig.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Author: Crownstone Team
+ * Copyright: Crownstone (https://crownstone.rocks)
+ * Date: Apr 26, 2022
+ * License: LGPLv3+, Apache License 2.0, and/or MIT (triple-licensed)
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+extern uint32_t g_HARDWARE_BOARD_ADDRESS;
```

Comment:
Comments should be in the header file (and can be removed from the source file).